docker-compose-dev.yml

```diff
@@ -1,4 +1,4 @@
-version: "3.7"
+version: "3"
```

Comment:
Why is it necessary to switch the version here but not in the release file?

Reply:
That change was staged to be commited by mistake. I was testing something between v3 and v3.7 and commited that by mistake. Will revert it now.